integration-tests/closed-connection/closed-connection.spec.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
```

Comment:
This is really a test for the `http` plugin. Can it easily be moved to the plugin tests? They are already integration tests, with the only difference that the test and the server run in the same process. If that's too difficult, maybe let's at least rename this folder to `http`.

Reply:
Moved, I didn't know that we have that kind of integration-test out of `integration-test` directory.